From 14ceb43dee3c3a029dd5398d809796011dd3d772 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 14 Apr 2020 10:52:53 -0400
Subject: [PATCH 1/1] demos/ppl_lpsol/examples: disable the boeing[12].mps
 tests.

These two tests fail due to numerical noise on 32-bit systems, and
this commit disables them to prevent build failures on Gentoo. This
one is a bit trickier than the other test removals because we need to
hack the expected output out of the expected_mpz file as well.

Warning: this is only half of the solution. The boeing[12].mps files
must still be deleted by the ebuild. The patch does not do this to
avoid running afoul of the 20KiB patch size limit in Gentoo.

Upstream-bug: https://www.cs.unipr.it/mantis/view.php?id=1078
Upstream-bug: https://www.cs.unipr.it/mantis/view.php?id=2113
---
 demos/ppl_lpsol/Makefile.am          |   21 -
 demos/ppl_lpsol/examples/Makefile.am |    2 -
 demos/ppl_lpsol/expected_mpz         | 1070 --------------------------
 3 files changed, 1093 deletions(-)

diff --git a/demos/ppl_lpsol/Makefile.am b/demos/ppl_lpsol/Makefile.am
index d6d6a38b2..7c73ba455 100644
--- a/demos/ppl_lpsol/Makefile.am
+++ b/demos/ppl_lpsol/Makefile.am
@@ -115,8 +115,6 @@ endif !ASSERTIONS_ENABLED
 TOUGH_EXAMPLES_FOR_ENUM = \
 adlittle.mps \
 blend.mps \
-boeing1.mps \
-boeing2.mps \
 egout.mps \
 kb2.mps \
 lseu.mps \
@@ -161,8 +159,6 @@ adlittle.mps \
 afiro.mps \
 bgprtr.mps \
 blend.mps \
-boeing1.mps \
-boeing2.mps \
 kb2.mps \
 sample.mps \
 sc105.mps \
@@ -172,13 +168,9 @@ ship08l.mps
 
 DO_NOT_OUTPUT_OPTIMUM_LOCATION_PROBLEMS = \
 adlittle.mps \
-boeing1.mps \
-boeing2.mps \
 rout.mps
 
 INDIVIDUALLY_CHECKED_PROBLEMS = \
-boeing1.mps \
-boeing2.mps \
 ex12.mps \
 mas74.mps \
 modglob.mps
@@ -291,19 +283,6 @@ check-local: ppl_lpsol$(EXEEXT)
 	  echo "*** ppl_lpsol -e -r $$m -oobtained $$file" >>obtained; \
 	  ./ppl_lpsol$(EXEEXT) -e -r $(CHECK_OPTION) $$m -oobtained $$path >/dev/null; \
 	done
-	$(ECHO_SIMPLEX_NOCHECK) "-r -m -c0.00000000518 boeing1.mps" >>obtained
-	-$(SIMPLEX_NOCHECK) -r -m -c0.00000000518 $(srcdir)/examples/boeing1.mps >/dev/null
-	$(ECHO_SIMPLEX_NOCHECK) "-r -M -c0.0000000105 boeing1.mps" >>obtained
-	-$(SIMPLEX_NOCHECK) -r -M -c0.0000000105 $(srcdir)/examples/boeing1.mps $$m >/dev/null
-	$(ECHO_SIMPLEX) "-n boeing1.mps" >>obtained
-	-$(SIMPLEX) -n $(srcdir)/examples/boeing1.mps >/dev/null
-	$(ECHO_SIMPLEX_NOCHECK) "-r -m -c0.00000000863 boeing2.mps" >>obtained
-	-$(SIMPLEX_NOCHECK) -r -m -c0.00000000863 $(srcdir)/examples/boeing2.mps >/dev/null
-	$(ECHO_SIMPLEX_NOCHECK) "-r -M -c0.00000000213 boeing2.mps" >>obtained
-	-$(SIMPLEX_NOCHECK) -r -M -c0.00000000213 $(srcdir)/examples/boeing2.mps $$m >/dev/null
-	$(ECHO_SIMPLEX) "-n boeing2.mps" >>obtained
-	-$(SIMPLEX) -n $(srcdir)/examples/boeing2.mps >/dev/null
-
 	$(ECHO_SIMPLEX) "-r -m mas74.mps" >>obtained
 	-$(SIMPLEX) -r -m $(srcdir)/examples/mas74.mps >/dev/null
 	$(ECHO_SIMPLEX_NOCHECK) "-r -M -c0.000123 mas74.mps" >>obtained
diff --git a/demos/ppl_lpsol/examples/Makefile.am b/demos/ppl_lpsol/examples/Makefile.am
index f621a68d4..ce333c7c5 100644
--- a/demos/ppl_lpsol/examples/Makefile.am
+++ b/demos/ppl_lpsol/examples/Makefile.am
@@ -26,8 +26,6 @@ adlittle.mps \
 afiro.mps \
 bgprtr.mps \
 blend.mps \
-boeing1.mps \
-boeing2.mps \
 egout.mps \
 ex12.mps \
 ex1.mps \
diff --git a/demos/ppl_lpsol/expected_mpz b/demos/ppl_lpsol/expected_mpz
index 2bb4f1658..5024da1f6 100644
--- a/demos/ppl_lpsol/expected_mpz
+++ b/demos/ppl_lpsol/expected_mpz
@@ -3937,1076 +3937,6 @@ Optimum value: 0
 Optimum location:
 x1 = 0
 x2 = 0
-*** ./ppl_lpsol -s -p1 -oobtained -r -m -c0.00000000518 boeing1.mps
-Optimum value: -335.2135675
-Optimum location:
-PBOSHNL0 = 0
-PBOSHNL1 = 12
-PBOSHNL7 = 0
-PBOSHNL8 = 0
-PBOSLAX0 = 14
-PBOSLAX1 = 0
-PBOSLAX7 = 0
-PBOSSEA0 = 0
-PBOSSEA1 = 0
-PBOSSEA2 = 49
-PBOSSFO0 = 60
-PBOSSFO1 = 73
-PBOSTPE1 = 1
-PBOSTPE2 = 0
-PBOSTYO1 = 3
-PBOSTYO2 = 0
-PBOSYUL0 = 28.13380282
-PBOSYUL1 = 347.7678621
-PBOSYUL2 = 0.325
-PBOSYUL3 = 336.7733351
-PBOSYUL4 = 0
-PBOSYVR0 = 0
-PBOSYVR1 = 0
-PBOSYVR2 = 26
-PBOSYWG0 = 0
-PBOSYWG1 = 0
-PBOSYWG7 = 37
-PBOSYYZ0 = 19.03169014
-PBOSYYZ1 = 195.9683099
-PBUROAK0 = 11
-PBUROAK1 = 16
-PBURSEA0 = 83
-PBURSEA1 = 0
-PBURSFO0 = 278
-PHNLLAX0 = 104.0482353
-PHNLLAX1 = 33.04
-PHNLLAX2 = 68.20588235
-PHNLLAX3 = 98
-PHNLLON0 = 0
-PHNLLON6 = 5
-PHNLPAR0 = 0
-PHNLPAR6 = 1
-PHNLSEA0 = 0
-PHNLSEA1 = 157.7943925
-PHNLSEA2 = 0
-PHNLSFO0 = 35
-PHNLSFO1 = 0
-PHNLSFO7 = 0
-PHNLYVR0 = 94.20560748
-PHNLYVR1 = 0
-PHNLYVR7 = 39.79439252
-PLASSEA0 = 70.01327253
-PLASSEA1 = 342.9867275
-PLASYVR0 = 0
-PLASYVR6 = 37
-PLAXOAK0 = 32
-PLAXOAK1 = 0
-PLAXOAK2 = 46
-PLAXSEA0 = 0
-PLAXSEA1 = 0
-PLAXSEA2 = 0
-PLAXSEA3 = 546.0147059
-PLAXSEA4 = 3.720588235
-PLAXSEA5 = 139
-PLAXSEA6 = 66.26744186
-PLAXSEA7 = 0
-PLAXSEA8 = 57
-PLAXSEA9 = 5.415868673
-PLAXSFO0 = 33.04
-PLAXSFO1 = 260.8488372
-PLAXSFO2 = 839.9632721
-PLAXSFO3 = 87.90697674
-PLAXSFO4 = 168
-PLAXSFO5 = 98
-PLAXSFO6 = 1464.240914
-PLAXTPE0 = 8.581395349
-PLAXTPE1 = 0
-PLAXTPE2 = 18
-PLAXTPE8 = 4.418604651
-PLAXTYO0 = 41
-PLAXTYO1 = 0
-PLAXTYO2 = 0
-PLAXTYO8 = 0
-PLAXYVR0 = 0
-PLAXYVR1 = 153.755814
-PLAXYVR2 = 39.24418605
-PLONPAR0 = 0
-PLONPAR1 = 0
-PLONPAR2 = 2
-PLONSEA0 = 0
-PLONSEA1 = 151
-PLONYVR0 = 0
-PLONYVR1 = 0
-PLONYVR7 = 51
-POAKONT0 = 0
-POAKONT1 = 13
-POAKSEA0 = 37.89108456
-POAKSEA1 = 77.15303309
-POAKSEA2 = 6.955882353
-PONTSFO0 = 177
-PONTSEA0 = 23.10431985
-PONTSEA1 = 24.89568015
-PONTSEA2 = 0
-PPARSEA0 = 0
-PPARSEA1 = 68
-PPARYVR0 = 0
-PPARYVR1 = 0
-PPARYVR7 = 24
-PRNOSEA0 = 0
-PRNOSEA1 = 291
-PRNOYVR0 = 67
-PSEASFO0 = 0
-PSEASFO1 = 839.9632721
-PSEASFO2 = 146.6809139
-PSEASFO3 = 236.6
-PSEASFO4 = 0
-PSEASFO5 = 50.75581395
-PSEASFO6 = 168
-PSEATPE0 = 15.81395349
-PSEATPE1 = 0
-PSEATPE2 = 0
-PSEATPE3 = 49.60465116
-PSEATYO0 = 59.98837209
-PSEATYO1 = 0
-PSEATYO2 = 0
-PSEATYO3 = 95.01162791
-PSEAYVR0 = 164.8598131
-PSEAYVR1 = 0
-PSEAYVR2 = 0
-PSEAYVR3 = 392
-PSEAYVR4 = 0
-PSEAYVR5 = 70.01327253
-PSEAYVR6 = 0
-PSEAYVR7 = 50.75581395
-PSEAYVR8 = 47.16549296
-PSFOTPE0 = 0
-PSFOTPE1 = 0
-PSFOTPE2 = 0
-PSFOTPE8 = 6
-PSFOTYO0 = 0
-PSFOTYO1 = 0
-PSFOTYO2 = 0
-PSFOTYO8 = 17
-PSFOYVR0 = 260.8488372
-PSFOYVR1 = 37.15116279
-PTPETYO0 = 111
-PTPETYO1 = 0
-PTPETYO2 = 0
-PTPETYO3 = 0
-PTPEYVR0 = 57
-PTYOYVR0 = 91
-PYULYVR0 = 36.4084507
-PYULYVR1 = 134.1343917
-PYULYVR2 = 139.6904909
-PYULYVR3 = 4.766666667
-PYULYWG0 = 0
-PYULYWG1 = 271.3172923
-PYULYWG2 = 0
-PYULYWG3 = 178.6827077
-PYULYYZ0 = 0
-PYULYYZ1 = 266.6818464
-PYULYYZ2 = 0
-PYULYYZ3 = 431.9057717
-PYULYYZ4 = 1913.412382
-PYVRYWG0 = 14.67957746
-PYVRYWG1 = 271.3172923
-PYVRYWG2 = 138.0031303
-PYVRYYZ0 = 4.352112676
-PYVRYYZ1 = 241.2835753
-PYVRYYZ2 = 126.3643121
-PYWGYYZ0 = 14.67957746
-PYWGYYZ1 = 138.0031303
-PYWGYYZ2 = 0
-PYWGYYZ3 = 125.3172923
-PBOSOAK0 = 11
-PBOSOAK6 = 0
-PBOSBUR1 = 7
-PBOSBUR2 = 0
-PBOSONT1 = 4
-PBOSONT2 = 0
-PBURYVR1 = 26
-PBURTYO1 = 2
-PBURTPE1 = 1
-PBURHNL0 = 11
-PBURHNL6 = 0
-PHNLOAK0 = 0
-PHNLOAK1 = 24
-PHNLOAK2 = 0
-PHNLOAK8 = 0
-PHNLONT0 = 9.705882353
-PHNLONT6 = 6.294117647
-PHNLYWG1 = 3
-PHNLYYZ1 = 24
-PHNLYUL1 = 40
-PLASTYO1 = 5
-PLASTPE1 = 1
-PLAXLON0 = 13
-PLAXLON6 = 0
-PLAXLON7 = 0
-PLAXPAR0 = 8
-PLAXPAR6 = 0
-PLAXPAR7 = 0
-PBURLON1 = 1
-PBURPAR1 = 1
-PLONONT1 = 1
-PLONOAK1 = 1
-POAKPAR1 = 2
-POAKTYO1 = 7
-POAKTPE1 = 2
-PONTPAR1 = 2
-PONTTYO1 = 2
-PONTTPE1 = 1
-PPARSFO1 = 2
-PRNOTYO1 = 5
-PRNOTPE1 = 2
-PTPEYWG1 = 21
-PTPEYYZ1 = 13
-PTPEYUL1 = 6
-PTYOYUL1 = 7
-PTYOYYZ1 = 17
-PTYOYWG1 = 25
-PLAXONT0 = 6.294117647
-GRDTIMO1 = 0
-GRDTIMN1 = -49.01286158
-GRDTIMO2 = 0
-GRDTIMN2 = 0
-GRDTIMO3 = 0
-GRDTIMN3 = -47
-GRDTIMO4 = 0
-GRDTIMN4 = 0
-GRDTIMO5 = 0
-GRDTIMN5 = -47.24353963
-GRDTIMO6 = 0
-GRDTIMN6 = -81
-N1001AC1 = 0
-N1001AC2 = 0
-N1001AC3 = 0
-N1002AC1 = 0
-N1002AC2 = 0
-N1002AC3 = 2.355140187
-N1003AC1 = 0
-N1003AC2 = 0.1603208556
-N1003AC3 = 0.7214438503
-N1004AC1 = 0.28
-N1004AC2 = 0
-N1004AC3 = 0
-N1005AC3 = 0.0859375
-N1105AC3 = 0.8382352941
-N1006AC3 = 0.07582720588
-N1007AC1 = 0
-N1007AC2 = 0
-N1007AC3 = 0
-N1008AC1 = 0
-N1008AC2 = 0
-N1008AC3 = 0
-N1008AC4 = 0
-N1008AC5 = 0
-N1008AC6 = 4.65801495
-N1009AC1 = 0
-N1009AC2 = 0.07488107822
-N1009AC3 = 0
-N1009AC4 = 0
-N1009AC5 = 1.921947674
-N1010AC1 = 0
-N1010AC2 = 0
-N1010AC3 = 0
-N1010AC4 = 0
-N1010AC5 = 0
-N1010AC6 = 7
-N1011AC1 = 0
-N1011AC2 = 3.085065996
-N1011AC3 = 0.6951730236
-N1011AC4 = 0
-N1011AC5 = 3.897217244
-N1011AC6 = 0
-N1012AC1 = 1.510832058
-N1012AC2 = 0
-N1012AC3 = 0
-N1012AC4 = 0
-N1012AC5 = 2.417331293
-N1013AC3 = 0
-N1013AC4 = 0
-N1013AC5 = 0
-N1013AC6 = 0.9241727941
-N1014AC3 = 0
-N1014AC4 = 0
-N1014AC5 = 1.691245404
-N1014AC6 = 0.2228170956
-N1015AC3 = 0
-N1015AC4 = 0.7048194147
-N1015AC5 = 0.1595430366
-N1015AC6 = 2.135637549
-N1016AC3 = 0
-N1016AC4 = 0
-N1016AC5 = 0.7
-N1016AC6 = 3.3
-N1017AC3 = 0
-N1017AC4 = 0
-N1017AC5 = 0
-N1017AC6 = 0.1617647059
-N1018AC1 = 0
-N1018AC2 = 0
-N1018AC3 = 0
-N1018AC4 = 0
-N1018AC5 = 0
-N1018AC6 = 1.196428571
-N1019AC1 = 0
-N1019AC2 = 0
-N1019AC3 = 0
-N1019AC4 = 0.2345133807
-N1019AC5 = 0.7654866193
-N1020AC1 = 0.2228332753
-N1020AC2 = 0
-N1020AC3 = 0
-N1020AC4 = 0
-N1020AC5 = 3.200696136
-N1020AC6 = 0
-N1021AC1 = 0.664202377
-N1021AC2 = 0
-N1021AC3 = 0
-N1021AC4 = 0
-N1021AC5 = 2.453858782
-N1022AC1 = 0
-N1023AC1 = 0
-N1026AC1 = 1
-N1027AC1 = 0.476744186
-N1028AC1 = 0
-N1029AC1 = 0.523255814
-N1030AC1 = 1
-N1032AC1 = 0
-N1032AC2 = 0
-N1032AC3 = 0
-N1032AC4 = 0.8274647887
-N1032AC5 = 0
-N1033AC1 = 0.4860357964
-N1033AC2 = 0.5004769224
-N1033AC3 = 0
-N1033AC4 = 0
-N1033AC5 = 2.061996183
-N1034AC1 = 0
-N1034AC2 = 0
-N1034AC3 = 3.174783885
-N1035AC1 = 0.6827679474
-N1035AC2 = 0
-N1035AC3 = 0
-N1035AC4 = 0.7269371829
-N1035AC5 = 0.2529831862
-N1036AC1 = 0.1083333333
-N1036AC2 = 0
-N1036AC3 = 0
-N1037AC4 = 0
-N1037AC5 = 0
-N1038AC4 = 0.2383968781
-N1038AC5 = 1.271450017
-N1039AC4 = 0
-N1039AC5 = 2.077705904
-N1040AC4 = 0
-N1040AC5 = 0
-N1040AC6 = 9.513243659
-N1041AC4 = 8.856901224
-N1041AC5 = 0
-N1041AC6 = 20
-N1042AC4 = 0
-N1042AC5 = 0
-N1042AC6 = 0
-N1043AC1 = 0.1725352113
-N1043AC2 = 0
-N1043AC3 = 0
-N1044AC1 = 0
-N1044AC2 = 0
-N1044AC3 = 1
-N1046AC3 = 1
-N1047AC1 = 0
-N1047AC2 = 0
-N1047AC3 = 1
-N1050AC3 = 0
-N1050AC4 = 0
-N1050AC5 = 0.0731874145
-N1051AC1 = 0
-N1051AC2 = 0
-N1051AC3 = 0
-N1051AC4 = 0
-N1051AC5 = 4.651904242
-N1051AC6 = 20
-*** ./ppl_lpsol -s -p1 -oobtained -r -M -c0.0000000105 boeing1.mps
-Optimum value: 286.9746573
-Optimum location:
-PBOSHNL0 = 0
-PBOSHNL1 = 10
-PBOSHNL7 = 0
-PBOSHNL8 = 0
-PBOSLAX0 = 0
-PBOSLAX1 = 0
-PBOSLAX7 = 12
-PBOSSEA0 = 0
-PBOSSEA1 = 40
-PBOSSEA2 = 0
-PBOSSFO0 = 48
-PBOSSFO1 = 73
-PBOSTPE1 = 0
-PBOSTPE2 = 0
-PBOSTYO1 = 0
-PBOSTYO2 = 0
-PBOSYUL0 = 372
-PBOSYUL1 = 224.0530973
-PBOSYUL2 = 0
-PBOSYUL3 = 11.94690265
-PBOSYUL4 = 0
-PBOSYVR0 = 0
-PBOSYVR1 = 23
-PBOSYVR2 = 0
-PBOSYWG0 = 33
-PBOSYWG1 = 0
-PBOSYWG7 = 0
-PBOSYYZ0 = 193
-PBOSYYZ1 = 0
-PBUROAK0 = 9
-PBUROAK1 = 15
-PBURSEA0 = 69
-PBURSEA1 = 0
-PBURSFO0 = 243
-PHNLLAX0 = 0
-PHNLLAX1 = 126.296875
-PHNLLAX2 = 100.03125
-PHNLLAX3 = 104.4375
-PHNLLON0 = 0
-PHNLLON6 = 0
-PHNLPAR0 = 0
-PHNLPAR6 = 0
-PHNLSEA0 = 0
-PHNLSEA1 = 135.2523364
-PHNLSEA2 = 0
-PHNLSFO0 = 0
-PHNLSFO1 = 0
-PHNLSFO7 = 31
-PHNLYVR0 = 80.74766355
-PHNLYVR1 = 0
-PHNLYVR7 = 35.25233645
-PLASSEA0 = 365
-PLASSEA1 = 0
-PLASYVR0 = 0
-PLASYVR6 = 33
-PLAXOAK0 = 42.765625
-PLAXOAK1 = 0
-PLAXOAK2 = 46
-PLAXSEA0 = 543.96875
-PLAXSEA1 = 0
-PLAXSEA2 = 0
-PLAXSEA3 = 0
-PLAXSEA4 = 6
-PLAXSEA5 = 139
-PLAXSEA6 = 0
-PLAXSEA7 = 0
-PLAXSEA8 = 66
-PLAXSEA9 = 12.03125
-PLAXSFO0 = 164.828125
-PLAXSFO1 = 842
-PLAXSFO2 = 0
-PLAXSFO3 = 168
-PLAXSFO4 = 168
-PLAXSFO5 = 144
-PLAXSFO6 = 1212.171875
-PLAXTPE0 = 0
-PLAXTPE1 = 18
-PLAXTPE2 = 9
-PLAXTPE8 = 0
-PLAXTYO0 = 0
-PLAXTYO1 = 0
-PLAXTYO2 = 0
-PLAXTYO8 = 36
-PLAXYVR0 = 116
-PLAXYVR1 = 0
-PLAXYVR2 = 57
-PLONPAR0 = 0
-PLONPAR1 = 0
-PLONPAR2 = 0
-PLONSEA0 = 0
-PLONSEA1 = 127
-PLONYVR0 = 0
-PLONYVR1 = 0
-PLONYVR7 = 45
-POAKONT0 = 0
-POAKONT1 = 11
-POAKSEA0 = 41
-POAKSEA1 = 57
-POAKSEA2 = 0
-PONTSFO0 = 155
-PONTSEA0 = 25
-PONTSEA1 = 12
-PONTSEA2 = 0
-PPARSEA0 = 0
-PPARSEA1 = 53
-PPARYVR0 = 0
-PPARYVR1 = 0
-PPARYVR7 = 21
-PRNOSEA0 = 255
-PRNOSEA1 = 0
-PRNOYVR0 = 60
-PSEASFO0 = 574
-PSEASFO1 = 0
-PSEASFO2 = 156
-PSEASFO3 = 224
-PSEASFO4 = 0
-PSEASFO5 = 168
-PSEASFO6 = 168
-PSEATPE0 = 0
-PSEATPE1 = 0
-PSEATPE2 = 42
-PSEATPE3 = 0
-PSEATYO0 = 0
-PSEATYO1 = 0
-PSEATYO2 = 126
-PSEATYO3 = 27
-PSEAYVR0 = 104.2523364
-PSEAYVR1 = 0
-PSEAYVR2 = 0
-PSEAYVR3 = 0
-PSEAYVR4 = 0
-PSEAYVR5 = 0
-PSEAYVR6 = 0
-PSEAYVR7 = 0
-PSEAYVR8 = 545
-PSFOTPE0 = 0
-PSFOTPE1 = 0
-PSFOTPE2 = 0
-PSFOTPE8 = 0
-PSFOTYO0 = 0
-PSFOTYO1 = 0
-PSFOTYO2 = 0
-PSFOTYO8 = 15
-PSFOYVR0 = 268
-PSFOYVR1 = 0
-PTPETYO0 = 0
-PTPETYO1 = 0
-PTPETYO2 = 0
-PTPETYO3 = 99
-PTPEYVR0 = 44
-PTYOYVR0 = 74
-PYULYVR0 = 182
-PYULYVR1 = 88
-PYULYVR2 = 0
-PYULYVR3 = 0
-PYULYWG0 = 161.3018868
-PYULYWG1 = 178
-PYULYWG2 = 31.69811321
-PYULYWG3 = 0
-PYULYYZ0 = 0
-PYULYYZ1 = 0
-PYULYYZ2 = 245.3018868
-PYULYYZ3 = 2104.698113
-PYULYYZ4 = 0
-PYVRYWG0 = 194.3018868
-PYVRYWG1 = 178
-PYVRYWG2 = 4.698113208
-PYVRYYZ0 = 328.6981132
-PYVRYYZ1 = 0
-PYVRYYZ2 = 4.301886792
-PYWGYYZ0 = 0
-PYWGYYZ1 = 4.698113208
-PYWGYYZ2 = 245.3018868
-PYWGYYZ3 = 0
-PBOSOAK0 = 9
-PBOSOAK6 = 0
-PBOSBUR1 = 0
-PBOSBUR2 = 0
-PBOSONT1 = 0
-PBOSONT2 = 0
-PBURYVR1 = 23
-PBURTYO1 = 0
-PBURTPE1 = 0
-PBURHNL0 = 9
-PBURHNL6 = 0
-PHNLOAK0 = 0
-PHNLOAK1 = 2.234375
-PHNLOAK2 = 0
-PHNLOAK8 = 18.765625
-PHNLONT0 = 0
-PHNLONT6 = 14
-PHNLYWG1 = 0
-PHNLYYZ1 = 21
-PHNLYUL1 = 35
-PLASTYO1 = 0
-PLASTPE1 = 0
-PLAXLON0 = 11
-PLAXLON6 = 0
-PLAXLON7 = 0
-PLAXPAR0 = 0
-PLAXPAR6 = 0
-PLAXPAR7 = 0
-PBURLON1 = 0
-PBURPAR1 = 0
-PLONONT1 = 0
-PLONOAK1 = 0
-POAKPAR1 = 0
-POAKTYO1 = 0
-POAKTPE1 = 0
-PONTPAR1 = 0
-PONTTYO1 = 0
-PONTTPE1 = 0
-PPARSFO1 = 0
-PRNOTYO1 = 0
-PRNOTPE1 = 0
-PTPEYWG1 = 18
-PTPEYYZ1 = 11
-PTPEYUL1 = 0
-PTYOYUL1 = 0
-PTYOYYZ1 = 15
-PTYOYWG1 = 22
-PLAXONT0 = 14
-GRDTIMO1 = 0
-GRDTIMN1 = -105
-GRDTIMO2 = 0
-GRDTIMN2 = -91
-GRDTIMO3 = 0
-GRDTIMN3 = -47
-GRDTIMO4 = 0
-GRDTIMN4 = -43.5
-GRDTIMO5 = 0
-GRDTIMN5 = -87
-GRDTIMO6 = 0
-GRDTIMN6 = -81
-N1001AC1 = 0
-N1001AC2 = 0
-N1001AC3 = 0
-N1002AC1 = 0
-N1002AC2 = 2.018691589
-N1002AC3 = 0
-N1003AC1 = 0
-N1003AC2 = 0
-N1003AC3 = 0
-N1004AC1 = 0
-N1004AC2 = 1.0703125
-N1004AC3 = 0
-N1005AC3 = 0.0703125
-N1105AC3 = 0.9296875
-N1006AC3 = 0
-N1007AC1 = 0
-N1007AC2 = 0
-N1007AC3 = 0
-N1008AC1 = 5.28156777
-N1008AC2 = 0
-N1008AC3 = 1.986798016
-N1008AC4 = 0
-N1008AC5 = 0
-N1008AC6 = 0
-N1009AC1 = 0
-N1009AC2 = 0
-N1009AC3 = 0
-N1009AC4 = 0
-N1009AC5 = 0
-N1010AC1 = 7
-N1010AC2 = 7
-N1010AC3 = 7
-N1010AC4 = 7
-N1010AC5 = 7
-N1010AC6 = 7
-N1011AC1 = 0
-N1011AC2 = 0
-N1011AC3 = 0
-N1011AC4 = 0
-N1011AC5 = 0
-N1011AC6 = 0
-N1012AC1 = 0
-N1012AC2 = 0
-N1012AC3 = 0
-N1012AC4 = 0
-N1012AC5 = 0
-N1013AC3 = 0
-N1013AC4 = 0
-N1013AC5 = 0
-N1013AC6 = 1
-N1014AC3 = 0
-N1014AC4 = 0
-N1014AC5 = 0
-N1014AC6 = 1.909090909
-N1015AC3 = 0
-N1015AC4 = 0
-N1015AC5 = 0
-N1015AC6 = 3
-N1016AC3 = 0
-N1016AC4 = 0
-N1016AC5 = 0
-N1016AC6 = 4
-N1017AC3 = 0
-N1017AC4 = 0
-N1017AC5 = 0
-N1017AC6 = 0.09090909091
-N1018AC1 = 3.782713663
-N1018AC2 = 6.749720385
-N1018AC3 = 7
-N1018AC4 = 0
-N1018AC5 = 4.039001305
-N1018AC6 = 0
-N1019AC1 = 0
-N1019AC2 = 1.412371134
-N1019AC3 = 0
-N1019AC4 = 2.587628866
-N1019AC5 = 0
-N1020AC1 = 0
-N1020AC2 = 0
-N1020AC3 = 0
-N1020AC4 = 0
-N1020AC5 = 0
-N1020AC6 = 0
-N1021AC1 = 0
-N1021AC2 = 0
-N1021AC3 = 0
-N1021AC4 = 0
-N1021AC5 = 0
-N1022AC1 = 0
-N1023AC1 = 0
-N1026AC1 = 1
-N1027AC1 = 0
-N1028AC1 = 0
-N1029AC1 = 1
-N1030AC1 = 1
-N1032AC1 = 1
-N1032AC2 = 1
-N1032AC3 = 1
-N1032AC4 = 1
-N1032AC5 = 1
-N1033AC1 = 0
-N1033AC2 = 1.371681416
-N1033AC3 = 0
-N1033AC4 = 0.6283185841
-N1033AC5 = 0
-N1034AC1 = 0
-N1034AC2 = 0
-N1034AC3 = 0
-N1035AC1 = 0
-N1035AC2 = 0.04608448823
-N1035AC3 = 0
-N1035AC4 = 0.01051928536
-N1035AC5 = 0
-N1036AC1 = 0
-N1036AC2 = 0
-N1036AC3 = 0
-N1037AC4 = 0
-N1037AC5 = 3.220930233
-N1038AC4 = 0
-N1038AC5 = 0
-N1039AC4 = 0
-N1039AC5 = 0
-N1040AC4 = 10
-N1040AC5 = 10
-N1040AC6 = 10
-N1041AC4 = 5.79377625
-N1041AC5 = 20
-N1041AC6 = 20
-N1042AC4 = 20
-N1042AC5 = 20
-N1042AC6 = 20
-N1043AC1 = 0
-N1043AC2 = 0
-N1043AC3 = 0
-N1044AC1 = 0
-N1044AC2 = 1
-N1044AC3 = 0
-N1046AC3 = 1
-N1047AC1 = 0
-N1047AC2 = 1
-N1047AC3 = 0
-N1050AC3 = 0.109375
-N1050AC4 = 0
-N1050AC5 = 0
-N1051AC1 = 20
-N1051AC2 = 20
-N1051AC3 = 20
-N1051AC4 = 0
-N1051AC5 = 20
-N1051AC6 = 11.52971853
-*** ./ppl_lpsol -s -p1 -c -oobtained -n boeing1.mps
-Feasible problem.
-*** ./ppl_lpsol -s -p1 -oobtained -r -m -c0.00000000863 boeing2.mps
-Optimum value: -315.018728
-Optimum location:
-PBOSORD0 = 302
-PBOSORD1 = 0
-PBOSORD2 = 0
-PBOSORD3 = 0
-PBOSORD4 = 0
-PBOSLGA0 = 712
-PBOSLGA1 = 268
-PBOSLGA2 = 0
-PBOSLGA3 = 1372
-PBOSCLE0 = 4.108024927
-PBOSCLE1 = 137.8919751
-PBOSCLE2 = 0
-PBOSCLE3 = 0
-PORDBOS0 = 34
-PORDBOS1 = 0
-PORDBOS2 = 0
-PORDBOS3 = 268
-PORDBOS4 = 0
-PORDLGA0 = 0
-PORDLGA1 = 515
-PORDLGA2 = 0
-PORDLGA3 = 0
-PORDCLE0 = 470.294686
-PORDCLE1 = 148.705314
-PORDCLE2 = 0
-PORDCLE3 = 0
-PLGABOS0 = 1162
-PLGABOS1 = 0
-PLGABOS2 = 458
-PLGABOS3 = 0
-PLGABOS4 = 0
-PLGABOS5 = 1123
-PLGAORD0 = 712
-PLGAORD1 = 0
-PLGAORD2 = 0
-PLGAORD3 = 0
-PLGACLE0 = 0
-PLGACLE1 = 249
-PLGACLE2 = 268
-PCLEBOS0 = 131
-PCLEBOS1 = 0
-PCLEBOS2 = 0
-PCLEBOS3 = 0
-PCLEBOS4 = 0
-PCLEORD0 = 178.5972891
-PCLEORD1 = 249
-PCLEORD2 = 16.40271092
-PCLEORD3 = 268
-PCLEORD4 = 0
-PCLELGA0 = 0
-PCLELGA1 = 0
-PCLELGA2 = 409
-PCLELGA3 = 0
-BBOSORD0 = 0
-BBOSORD1 = 0
-BBOSCLE0 = 0.1616272102
-BBOSCLE1 = 7.026778587
-BORDBOS0 = 0
-CBOSORD0 = 12
-CBOSORD1 = 0
-CBOSORD2 = 0
-CBOSORD3 = 0
-CBOSORD4 = 0
-CBOSCLE0 = 0
-CBOSCLE1 = 8.811594203
-CBOSCLE2 = 0
-CBOSCLE3 = 0
-CORDBOS0 = 1.5
-CORDBOS1 = 22.5
-CORDBOS2 = 0
-CORDBOS3 = 0
-CORDBOS4 = 0
-CORDLGA0 = 13
-CORDLGA1 = 0
-CORDLGA2 = 0
-CORDLGA3 = 0
-CLGAORD0 = 22.5
-CLGAORD1 = 22.5
-CLGAORD2 = 0
-CLGAORD3 = 0
-CLGACLE0 = 16
-CLGACLE1 = 0
-CLGACLE2 = 0
-CCLELGA0 = 0
-CCLELGA1 = 5
-CCLELGA2 = 0
-CCLELGA3 = 0
-GRDTIMO1 = 0
-GRDTIMN1 = 0
-GRDTIMO2 = 0
-GRDTIMN2 = -76.79770851
-GRDTIMN3 = -10.27177739
-GRDTIMN4 = -21.60844089
-N1003AC1 = 0
-N1003AC2 = 6.724489796
-N1003AC3 = 0.4
-N1003AC4 = 0
-N1005AC1 = 0.01346893419
-N1005AC2 = 0
-N1005AC3 = 0
-N1005AC4 = 0
-N1011AC1 = 0.5855648822
-N1011AC2 = 0
-N1011AC3 = 0.2937198068
-N1011AC4 = 0
-N1013AC2 = 7.265306122
-N1013AC4 = 2
-N1015AC2 = 0
-N1015AC4 = 1.422222222
-N1017AC2 = 0
-N1017AC4 = 2
-N1019AC2 = 2.540816327
-N1019AC4 = 0
-N1021AC1 = 0.01346893419
-N1021AC2 = 0.1254559795
-N1021AC3 = 0
-N1021AC4 = 0
-N1002AC1 = 0
-N1002AC2 = 4.798925367
-N1002AC3 = 0
-N1002AC4 = 0
-N1004AC2 = 0
-N1004AC4 = 2
-N1006AC1 = 0.5990338164
-N1006AC2 = 0
-N1006AC3 = 0.6937198068
-N1006AC4 = 0
-N1008AC2 = 11.85714286
-N1008AC4 = 2
-N1010AC2 = 0
-N1010AC4 = 0
-N1012AC2 = 0
-N1012AC4 = 1.422222222
-N1014AC2 = 4.673469388
-N1014AC4 = 0
-N1100AC2 = 2.734693878
-N1100AC4 = 0
-N1102AC2 = 0
-N1102AC4 = 0
-N1200AC2 = 14
-N1200AC4 = 0
-N1201AC2 = 11.45918367
-N1201AC4 = 0
-*** ./ppl_lpsol -s -p1 -oobtained -r -M -c0.00000000213 boeing2.mps
-Optimum value: -73.3689691
-Optimum location:
-PBOSORD0 = 28
-PBOSORD1 = 0
-PBOSORD2 = 0
-PBOSORD3 = 0
-PBOSORD4 = 249
-PBOSLGA0 = 258.2518986
-PBOSLGA1 = 686
-PBOSLGA2 = 35.74810145
-PBOSLGA3 = 1372
-PBOSCLE0 = 0
-PBOSCLE1 = 0
-PBOSCLE2 = 0
-PBOSCLE3 = 142
-PORDBOS0 = 302
-PORDBOS1 = 0
-PORDBOS2 = 0
-PORDBOS3 = 0
-PORDBOS4 = 0
-PORDLGA0 = 0
-PORDLGA1 = 0
-PORDLGA2 = 515
-PORDLGA3 = 0
-PORDCLE0 = 0
-PORDCLE1 = 619
-PORDCLE2 = 0
-PORDCLE3 = 0
-PLGABOS0 = 0
-PLGABOS1 = 712
-PLGABOS2 = 0
-PLGABOS3 = 686
-PLGABOS4 = 426.7481014
-PLGABOS5 = 918.2518986
-PLGAORD0 = 258.2518986
-PLGAORD1 = 0
-PLGAORD2 = 453.7481014
-PLGAORD3 = 0
-PLGACLE0 = 517
-PLGACLE1 = 0
-PLGACLE2 = 0
-PCLEBOS0 = 131
-PCLEBOS1 = 0
-PCLEBOS2 = 0
-PCLEBOS3 = 0
-PCLEBOS4 = 0
-PCLEORD0 = 0
-PCLEORD1 = 0
-PCLEORD2 = 712
-PCLEORD3 = 0
-PCLEORD4 = 0
-PCLELGA0 = 197
-PCLELGA1 = 70
-PCLELGA2 = 0
-PCLELGA3 = 142
-BBOSORD0 = 0
-BBOSORD1 = 0
-BBOSCLE0 = 0
-BBOSCLE1 = 0
-BORDBOS0 = 0
-CBOSORD0 = 0
-CBOSORD1 = 0
-CBOSORD2 = 0
-CBOSORD3 = 0
-CBOSORD4 = 0
-CBOSCLE0 = 0
-CBOSCLE1 = 0
-CBOSCLE2 = 12.8
-CBOSCLE3 = 0
-CORDBOS0 = 19.2
-CORDBOS1 = 0
-CORDBOS2 = 0
-CORDBOS3 = 0
-CORDBOS4 = 0
-CORDLGA0 = 0
-CORDLGA1 = 0
-CORDLGA2 = 10.4
-CORDLGA3 = 0
-CLGAORD0 = 0
-CLGAORD1 = 0
-CLGAORD2 = 10.4
-CLGAORD3 = 25.6
-CLGACLE0 = 12.8
-CLGACLE1 = 0
-CLGACLE2 = 0
-CCLELGA0 = 0
-CCLELGA1 = 0
-CCLELGA2 = 0
-CCLELGA3 = 0
-GRDTIMO1 = 73.01076
-GRDTIMN1 = -100
-GRDTIMO2 = 0
-GRDTIMN2 = -90
-GRDTIMN3 = -45
-GRDTIMN4 = -45
-N1003AC1 = 7
-N1003AC2 = 0
-N1003AC3 = 0.6856076556
-N1003AC4 = 0
-N1005AC1 = 0
-N1005AC2 = 0
-N1005AC3 = 0
-N1005AC4 = 0
-N1011AC1 = 0
-N1011AC2 = 0
-N1011AC3 = 1.314392344
-N1011AC4 = 0
-N1013AC2 = 2.635223455
-N1013AC4 = 0
-N1015AC2 = 5.275510204
-N1015AC4 = 2
-N1017AC2 = 0
-N1017AC4 = 0
-N1019AC2 = 4.630082668
-N1019AC4 = 0.9244444444
-N1021AC1 = 7
-N1021AC2 = 0
-N1021AC3 = 2
-N1021AC4 = 2
-N1002AC1 = 7
-N1002AC2 = 0
-N1002AC3 = 2
-N1002AC4 = 2
-N1004AC2 = 0
-N1004AC4 = 0
-N1006AC1 = 7
-N1006AC2 = 0
-N1006AC3 = 2
-N1006AC4 = 0
-N1008AC2 = 0
-N1008AC4 = 0
-N1010AC2 = 7.265306122
-N1010AC4 = 0.9244444444
-N1012AC2 = 5.275510204
-N1012AC4 = 2
-N1014AC2 = 0
-N1014AC4 = 0
-N1100AC2 = 7
-N1100AC4 = 5.274621909
-N1102AC2 = 4.354572464
-N1102AC4 = 0
-N1200AC2 = 14
-N1200AC4 = 0.9244444444
-N1201AC2 = 9.369917332
-N1201AC4 = 0
-*** ./ppl_lpsol -s -p1 -c -oobtained -n boeing2.mps
-Feasible problem.
 *** ./ppl_lpsol -s -p1 -c -oobtained -r -m mas74.mps
 Optimum value: 10482.79528
 Optimum location:
-- 
2.24.1

